tasks/create-product-sbom/tests/test-create-product-sbom-basic.yaml

```diff
@@ -0,0 +1,109 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-create-product-sbom-basic
+spec:
+  description: |
+    Create a product-level SBOM where components contain only one purl.
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup
+            image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
```

Comment:
Can you update the tests to use the same image as the task? It's not strictly required, but it's a convention we have.

Reply:
Done